solidity/test/support/helpers.js

```diff
@@ -95,10 +94,11 @@ export const getLatestEvent = async (contract) => {
 
 export const requestDataFrom = (oc, link, amount, args, options) => {
   if (!options) options = {}
-  return link.transferAndCall(oc.address, amount, args, options)
+  return link.transferAndCall( oc.address, amount.toString(), args, options)
```

Comment:
Does a leading whitespace pass our linter? Let's remove that.

Reply:
Thanks. 53a65ed2